lib/Widgets/OutgoingTransmissionListEntry.vala

```diff
@@ -64,14 +69,18 @@ public class Drop.Widgets.OutgoingTransmissionListEntry : TransmissionListEntry
         transmission.state_changed.connect (display_state);
         transmission.progress_changed.connect (display_progress);
 
-        cancel_button.clicked.connect (() => {
-            try {
-                transmission.cancel ();
-                this.destroy ();
-            } catch (Error e) {
-                stderr.printf ("Could not cancel transmition: %s", e.message);
-            }
-        });
+        cancel_button.clicked.connect (cancel);
+
```

Comment:
you don't need this blank line, because these two signal connections are nearly the same.